run.sh

```diff
@@ -14,9 +14,15 @@ PROJECT_NAME=$USER-${PWD##*/}
 docker compose -p $PROJECT_NAME down -v
 
 # Get relay chain spec, genesis wasm+head
-
 docker run --rm $KILT_IMG export-genesis-state --chain=$KILT_RAW_SPEC_FILE --runtime=$KILT_RUNTIME > specs/kilt-genesis.hex
 docker run --rm $KILT_IMG export-genesis-wasm --chain=$KILT_RAW_SPEC_FILE --runtime=$KILT_RUNTIME > specs/kilt.wasm
-docker run --rm --entrypoint cat $KILT_IMG /node/dev-specs/kilt-parachain/peregrine-relay.json > specs/polkadot.raw.json
+docker run --rm --entrypoint cat $KILT_IMG /node/dev-specs/kilt-parachain/peregrine-stg-relay.json > specs/polkadot.raw.json
+
+# Specs cannot be accessed from parent directories when building the docker image, thus they need to be copied
+cp specs/kilt-genesis.hex scripts/registerParachain/kilt-genesis.hex
+cp specs/kilt.wasm scripts/registerParachain/kilt.wasm
 
-docker compose -p $PROJECT_NAME up -d
\ No newline at end of file
+# Remove nodejs image in case it is not rebuilt because of cache
+docker compose up --build --force-recreate
```

Comment:
I'm unsure whether we actually want to commit the `package-lock.json` in this case because otherwise we always build the node image for ~10s.

Reply:
why would we always build the image? the `package-lock.json` will only update if we have upgrade dependencies right?